pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -214,6 +214,47 @@ nixBuild() {
   fi
 }
 
+nixFlakeBuild() {
+    if [ -z "$buildHost" ]; then
+        nix build "$@" --out-link "${tmpDir}/result"
+        readlink -f "${tmpDir}/result"
+    else
+        local attr="$1"
+        shift 1
+        local evalArgs=()
+        local buildArgs=()
+        while [ "$#" -gt 0 ]; do
+            local i="$1"; shift 1
+            case "$i" in
+              --recreate-lock-file|--no-update-lock-file|--no-write-lock-file|--no-registries|--commit-lock-file)
+                evalArgs+=("$i")
+                ;;
+              --update-input)
+                local j="$1"; shift 1
+                evalArgs+=("$i" "$j")
+                ;;
+              --override-input)
+                local j="$1"; shift 1
+                local k="$1"; shift 1
+                evalArgs+=("$i" "$j" "$k")
+                ;;
+              *)
+                buildArgs+=("$i")
+                ;;
+            esac
```

Comment:
```suggestion
            case "$1" in
              --recreate-lock-file|--no-update-lock-file|--no-write-lock-file|--no-registries|--commit-lock-file)
                evalArgs+=("$1")
                ;;
              --update-input)
                evalArgs+=("$1" "$2")
                shift 1
                ;;
              --override-input)
                evalArgs+=("$1" "$2" "$3")
                shift 2
                ;;
              *)
                buildArgs+=("$1")
                ;;
            esac
            shift 1
```

Reply:
I used the style of `nixBuild` above. I'm not necessarily against this change, but it should be done for both if it is applied, IMO.